src/XrdTpc/XrdTpcTPC.cc

```diff
@@ -442,28 +439,32 @@ int TPCHandler::DetermineXferSize(CURL *curl, XrdHttpExtReq &req, State &state,
                                   bool &success, TPCLogRecord &rec, bool shouldReturnErrorToClient) {
     success = false;
     curl_easy_setopt(curl, CURLOPT_NOBODY, 1);
+    // Set a custom timeout of 120 seconds (= CONNECT_TIMEOUT for convenience) for the HEAD request
+    curl_easy_setopt(curl, CURLOPT_TIMEOUT, CONNECT_TIMEOUT);
```

Comment:
Is there a reason why CONNECT_TIMEOUT is not configurable? Just wondering.

Reply:
I believe it's because we've set it to 60 seconds (Linux default connection timeout) : https://github.com/xrootd/xrootd/pull/2242#discussion_r1588798785
If there's a need to make it configurable, I can of course, tell me :)

Reply:
No I tdon't think we need to make it configurable at this point. Let's see if we get any complaints.